packages/server/src/oauth/token.test.ts

```diff
@@ -1472,7 +1542,7 @@ describe('OAuth2 Token', () => {
       client_assertion: jwt,
     });
     expect(res.status).toBe(200);
-    expect(jwtVerify).toHaveBeenCalledTimes(3);
+    expect(jwtVerify).toHaveBeenCalledTimes(4);
```

Comment:
Out of curiosity, why did this change from 3 to 4?

Reply:
Due to [this additional call](https://github.com/medplum/medplum/pull/5918/files#diff-8684369c3fa991aac52183ced98bbf3cf1b849be67da8c3e4772d9db132f098aR609) to `jwtVerify()`